Automattic/pocket-casts-android
# Display Longest Episode story

## Description

This adds total time story UI.

I had to drop "and" between different length sections due to localization nightmare. So "2 days and 10 hours" is "2 days 10 hours" instead and so on.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3054_3925

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the longest episode story and verify it.

## Screenshots or Screencast 

### Preview

| Regular | Small |
| - | - |
| ![Screenshot_20241017-125610 (1)](https://github.com/user-attachments/assets/0f9974a5-0cb8-453d-a863-2ee07fa5435e) | ![smol](https://github.com/user-attachments/assets/42b88f54-0b37-4323-a4d3-20a23ea89e97) |

### Animation

https://github.com/user-attachments/assets/a1e50990-f336-4237-b721-50676933d6a2

## Checklist
 
#### I have tested any UI changes...
